<?xml version="1.0" ?>

<launch>
  <arg name="ph_port"                default="/dev/ph_sensor"/>
  <arg name="ph_baud"                default="9600"/>
  <arg name="ph_publish_voltage"     default="false"/>

  <arg name="pipetty_motor_port"     default="/dev/pipetty_motor"/>
  <arg name="pipetty_motor_baud"     default="9600"/>

  <arg name="pipetty_port"           default="/dev/pipetty_pro"/>
  <arg name="pipetty_baud"           default="31520"/>

  <group ns="robot_control">
    <!-- ph_sensor -->
    <include file="$(find lite6_labauto)/launch/ph_sensor.launch">
      <arg name="port"            value="$(arg ph_port)" />
      <arg name="baud"            value="$(arg ph_baud)" />
      <arg name="publish_voltage" value="$(arg ph_publish_voltage)" />
    </include>

    <!-- pipetty node -->
    <include file="$(find auto_pipetty)/launch/auto_pipetty.launch">
      <arg name="pipetty_port"       value="$(arg pipetty_port)" />
      <arg name="pipetty_baud"       value="$(arg pipetty_baud)" />
      <arg name="pipetty_motor_port" value="$(arg pipetty_motor_port)" />
      <arg name="pipetty_motor_baud" value="$(arg pipetty_motor_baud)" />
    </include>
  </group>
</launch>
